### PR TITLE
Templatization of the product edit form

### PR DESCRIPTION
**Pull request by stephanegigandet**
@roshnaeem has templatized the code for the product edit form. This form is very complex, so please help check it!
This PR is currently live on the test server: https://world.openfoodfacts.dev

### CODE REVIEW

File: templates/web/pages/product_edit/display_input_field.tt.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+<label for="[% field %]">
+    [% fieldtype %]
+    [% IF language.defined %]
+        (<span class="tab_language">[% language %]</span>)
+    [% END %]
+</label>
+
+[% IF field.match('infocard') || field.match('^packaging_text') %]
+    <textarea name="[% field %]" id="[% field %]" lang="${display_lc}">[% value %]</textarea>
```
Comment (roshnaeem) [inline templates/web/pages/product_edit/display_input_field.tt.html:11]: I forgot to change it. Can you please check this `lang="${display_lc}"`

**Review by roshnaeem — CHANGES_REQUESTED** [review]

**Comment by roshnaeem** [comment]
Can you please check if changing language in one tab changes the language in other tabs as well?

**Comment by stephanegigandet** [comment]
> Also, two `data-alert` divs are missing. I guess the js file `product_multilingual.js` code is not working on my end.

Sorry I don't understand, which data-alert divs are missing?

**Comment by roshnaeem** [comment]
> > Also, two `data-alert` divs are missing. I guess the js file `product_multilingual.js` code is not working on my end.
> 
> Sorry I don't understand, which data-alert divs are missing?

Screenshot from 2021-07-27 14-58-14

File: templates/web/pages/product_edit/product_edit_form_display.tt.html [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+            <input type="text" name="new_code" id="new_code" class="text" value="" /><br/>
+        [% END %]
+
+        [% IF moderator || owner_id %]
+            <input type="checkbox" id="obsolete" name="obsolete" [% obsolete_checked %]/>
+            <label for="obsolete" class="checkbox_label">[% lang("obsolete") %]</label>
+            [% display_field_obsolete %]
+        [% END %]
+
+        <div data-alert class="alert-box info store-state" id="warning_3rd_party_content" style="display:none;">
```
Comment (roshnaeem) [inline templates/web/pages/product_edit/product_edit_form_display.tt.html:33]: js file is supposed to change the display.

File: templates/web/pages/product_edit/display_input_field.tt.html (hunk 1 above)
Comment (stephanegigandet) [reply templates/web/pages/product_edit/display_input_field.tt.html:11]: It seems to work. It wasn't called for ingredients_text though. I changed it to use the display_field function also for ingredients_text

**Review by roshnaeem — APPROVED** [review]

**Pull request merged**